added background animation with pure css;fixed overflow bug

Milosavljevichh opened the pull request:
Note: Feedback needed on the implemented animation, option for toggling the animation hasn't been implemented yet because I'm waiting for your feedback.

Milosavljevichh committed:
added background animation with pure css

vinaytheprogrammer commented:
go for it
![image](https://github.com/vinaytheprogrammer/Chitchat/assets/49182318/05dc48c6-89f8-4c85-b6a0-e9893e148818)

if you resolve overflow issue go for it. make a toggle

Milosavljevichh committed:
fixed overflow bug

Milosavljevichh committed:
Added animation toggler

Milosavljevichh committed:
stylized animation toggle button

Milosavljevichh commented:
> go for it ![image](https://private-user-images.githubusercontent.com/49182318/344574806-05dc48c6-89f8-4c85-b6a0-e9893e148818.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk4NDIyNTEsIm5iZiI6MTcxOTg0MTk1MSwicGF0aCI6Ii80OTE4MjMxOC8zNDQ1NzQ4MDYtMDVkYzQ4YzYtODlmOC00Yzg1LWI2YTAtZTk4OTNlMTQ4ODE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzAxVDEzNTIzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc0N2M1ZmEzODdmZDI2NTIxNzM4ZmM1YmFmYTlhMjM3ZjRjMjVjNzViMDg0MTVmZjQ3YzYzNTUwM2FkMTA2NmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ORgQwp35xpf7RBaha-wfGLqg6WGngp--u99wzENtCvw)
> 
> if you resolve overflow issue go for it. make a toggle

fixed overflowing issue and added animation toggle button. Because of the recent commits on this project, my branch has some conflicts which I'll resolve soon.

Milosavljevichh committed:
Merge branch 'main' into backgroundAnimation
